src/workerd/server/server.c++

```diff
@@ -1392,6 +1541,25 @@ public:
       KJ_LOG(ERROR, exception);
```

Comment:
I think this `taskFailed()` is no longer used since we don't have any `TaskSet`s anymore. Can delete?